### PR TITLE
Hawkular APM: the future

### PR DESCRIPTION
**Pull request by objectiser**

### CODE REVIEW

**Comment by objectiser** [comment]
@jpkrohling @pavolloffay @theute @pilhuhn Initial draft - not to be merged yet. Comments welcome.

File: src/main/jbake/content/blog/2017/04/07/hawkular-apm-jaeger.adoc [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+Gary Brown
+2017-04-07
+:jbake-type: post
+:jbake-status: published
+:jbake-tags: blog, apm, opentracing, jaeger
+
+Since October last year, the Hawkular APM project has been actively working on supporting, and contributing to,
+the http://opentracing.io/[OpenTracing] standard.
+
+We began by implementing our owh Java OpenTracing provider, followed by a Node.js/Javascript implementation. We have
```
Comment (jpkrohling) [inline src/main/jbake/content/blog/2017/04/07/hawkular-apm-jaeger.adoc]: `s/owh/own`

**Review by jpkrohling — COMMENTED** [review]
Other than the typo, it looks great. I'd add some more on the benefits of collaboration (faster innovation? their internal use of Jaeger tracing N microservices at Uber?)

File: src/main/jbake/content/blog/2017/04/07/hawkular-apm-jaeger.adoc (hunk 1 above)
Comment (objectiser) [reply src/main/jbake/content/blog/2017/04/07/hawkular-apm-jaeger.adoc]: Must buy a spell checker :)

**Comment by pavolloffay** [comment]
LGTM :+1:  (shouldn't this happen in parallel with Uber announcement?)

**Comment by objectiser** [comment]
@pavolloffay Yes that is the plan - so just getting it ready to push the button at the appropriate time.

**Pull request merged**